source/agora/node/Ledger.d

```diff
@@ -772,60 +772,52 @@ public class Ledger
         const Scalar challenge = hashFull(block);
         const enrolled_validators = this.enroll_man.getCountOfValidators(block.header.height);
 
-        log.trace("Validators expected to sign this block is {}", enrolled_validators);
         // Check that more than half have signed
         auto signed = iota(0, enrolled_validators).filter!(i => block.header.validators[i]).count();
         if (signed <= enrolled_validators / 2)
         {
-            log.error("Only {} signed. Require more than {} out of {} validators to sign for externalizing slot height {}.",
-                      signed, enrolled_validators / 2, enrolled_validators, block.header.height);
-            return "Only " ~ to!string(signed) ~ " signed the block. Require more than " ~ to!string(enrolled_validators);
+            // Can't use '51%' here because 50.000001% is enough
```

Comment:
I don't see the point of this comment.

Reply:
True. Originally I wrote something like "Less than 51% signatures", but that could have been misleading.

Reply:
We could also just go with the `51%` anyway given how much of a corner case it is. "Block does not have absolute majority of signature" also comes to mind.

Reply:
I think it is ok as is but just without the comment.

Reply:
Removed this line